subsys/bluetooth/host/conn.c

```diff
@@ -530,6 +530,11 @@ static int send_acl(struct bt_conn *conn, struct net_buf *buf, uint8_t flags)
 	return bt_send(buf);
 }
 
+static enum bt_iso_timestamp contains_iso_timestamp(struct net_buf *buf)
+{
+	return net_buf_headroom(buf) == BT_BUF_ISO_SIZE(0) - sizeof(struct bt_hci_iso_sdu_ts_hdr);
```

Comment:
You're essentially implicitly converting `bool` to `enum bt_iso_timestamp` here. I think it'd be cleaner to do that conversion explicitly in the function and return an actual enum value.